### PR TITLE
Add defaultExpose option to marathon section

### PR DESCRIPTION
**Pull request by sample**
defaultExpose for marathon tasks, idea from https://github.com/containous/traefik/issues/233

### CODE REVIEW

File: cmd.go [hunk 1]
```diff
@@ -126,6 +126,7 @@ func init() {
 	traefikCmd.PersistentFlags().StringVar(&arguments.Marathon.Filename, "marathon.filename", "", "Override default configuration template. For advanced users :)")
 	traefikCmd.PersistentFlags().StringVar(&arguments.Marathon.Endpoint, "marathon.endpoint", "http://127.0.0.1:8080", "Marathon server endpoint. You can also specify multiple endpoint for Marathon")
 	traefikCmd.PersistentFlags().StringVar(&arguments.Marathon.Domain, "marathon.domain", "", "Default domain used")
+	traefikCmd.PersistentFlags().BoolVar(&arguments.Marathon.DefaultExpose, "marathon.defaultexpose", true, "Expose Marathon apps by default")
```
Comment (emilevauge) [inline cmd.go]: should be `marathon.defaultExpose` to stay homogeneous ;)

**Comment by emilevauge** [comment]
Thanks a lot @sample ! This is great.
I have only minor remark (see my code comment).
We should also add this to the Docker provider ;)

**Comment by emilevauge** [comment]
Could you rebase and squash your commits?
Other than that LGTM!
/cc @vdemeester

File: provider/marathon.go [hunk 2]
```diff
@@ -150,7 +151,10 @@ func taskFilter(task marathon.Task, applications *marathon.Applications) bool {
 		log.Errorf("Unable to get marathon application from task %s", task.AppID)
 		return false
 	}
-	if application.Labels["traefik.enable"] == "false" {
+	if !defaultExposeFlag && application.Labels["traefik.enable"] != "true" {
```
Comment (vdemeester) [inline provider/marathon.go]: This looks a little bit complicated to read. We could maybe use something like : 

```go
if !isApplicationEnabled(application) {
    // […]
    return false
}

// […]

func isApplicationEnabled(application *marathon.Application, exposedByDefault bool) bool {
    return exposedByDefault && application.Labels["traefik.enable"] != "false" || application.Labels["traefik.enable"] == "true"
}
```

File: provider/marathon.go [hunk 3]
```diff
@@ -20,6 +20,7 @@ type Marathon struct {
 	BaseProvider   `mapstructure:",squash"`
 	Endpoint       string
 	Domain         string
+	DefaultExpose  bool
```
Comment (vdemeester) [inline provider/marathon.go]: I would tend to prefer `ExposedByDefault` than `DefaultExpose` but that's just a nit I think.

**Comment by emilevauge** [comment]
Thanks @vdemeester for your review.
@sample, your rebase seems strange (you carried other commits) ;)

**Comment by sample** [comment]
@emilevauge looks better now)

**Comment by emilevauge** [comment]
Thanks @sample, @vdemeester made some minor comments ;)

**Comment by sample** [comment]
@emilevauge can I do something else for this PR?

**Comment by emilevauge** [comment]
Could you rebase on master please ?
@vdemeester OK for you ?

**Comment by vdemeester** [comment]
LGTM 🐼

**Comment by emilevauge** [comment]
Thanks a lot @sample :)

**Pull request merged**